### PR TITLE
Add custom config

### PR DESCRIPTION
**Pull request by aashu**

### CODE REVIEW

File: 11/aashu.config [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# NikGapps configuration file
```
Comment (googlabati) [inline 11/aashu.config:1]: /home/rehan/Downloads/rehangapps.config

File: 11/aashu.config [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+# visit https://nikgapps.com/misc/2020/11/22/NikGapps-Config.html to read everything about nikgapps
+
+# If you are not sure about the config, just skip making changes to it or comment it by adding # before it
+
+InstallPartition=product
```
Comment (googlabati) [inline 11/aashu.config:7]: product

File: 11/aashu.config [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+mode=install
+
+# set WipeDalvikCache=0 if you don't want the installer to wipe dalvik/cache after installing the gapps
```
Comment (googlabati) [inline 11/aashu.config:11]: wipes

File: 11/aashu.config [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+WipeDalvikCache=1
```
Comment (googlabati) [inline 11/aashu.config:12]: 1

File: 11/aashu.config [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+# set WipeRuntimePermissions=1 if you want to wipe runtime permissions
+WipeRuntimePermissions=0
+
+# Addon.d configs set them to 0 to skip the functionality
+execute.d=1
+mount.d=1
```
Comment (googlabati) [inline 11/aashu.config:19]: mount

File: 11/aashu.config [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+unmount.d=1
+
+# Addon version (set it to 3 if you use lineageOs recovery, set it to 2 if you're using twrp recovery)
+addon_version.d=3
+
+# Following are the packages with default configuration
+# Set Core=0 if you want to skip installing all packages belonging to Core Package
+Core=1
```
Comment (googlabati) [inline 11/aashu.config:27]: core

File: 11/aashu.config [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+>>ExtraFiles=1
```
Comment (googlabati) [inline 11/aashu.config:28]: files

File: 11/aashu.config [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+>>GooglePlayStore=1
```
Comment (googlabati) [inline 11/aashu.config:29]: playstore

File: 11/aashu.config [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+>>GoogleServicesFramework=1
```
Comment (googlabati) [inline 11/aashu.config:30]: play services

File: 11/aashu.config [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+>>GoogleContactsSyncAdapter=1
```
Comment (googlabati) [inline 11/aashu.config:31]: contacts

File: 11/aashu.config [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+>>GoogleCalendarSyncAdapter=1
```
Comment (googlabati) [inline 11/aashu.config:32]: calendar

File: 11/aashu.config [hunk 12]
```diff
@@ -0,0 +1,125 @@
...
+>>GmsCore=1
```
Comment (googlabati) [inline 11/aashu.config:33]: 1

File: 11/aashu.config [hunk 13]
```diff
@@ -0,0 +1,125 @@
...
+
+DigitalWellbeing=1
+VancedManager=1
+GoogleMessages=0
```
Comment (googlabati) [inline 11/aashu.config:37]: messeges

File: 11/aashu.config [hunk 14]
```diff
@@ -0,0 +1,125 @@
...
+GoogleDialer=1
```
Comment (googlabati) [inline 11/aashu.config:38]: phone

File: 11/aashu.config [hunk 15]
```diff
@@ -0,0 +1,125 @@
...
+GoogleContacts=1
```
Comment (googlabati) [inline 11/aashu.config:39]: contacts

File: 11/aashu.config [hunk 16]
```diff
@@ -0,0 +1,125 @@
...
+CarrierServices=0
```
Comment (googlabati) [inline 11/aashu.config:40]: services

File: 11/aashu.config [hunk 17]
```diff
@@ -0,0 +1,125 @@
...
+GoogleClock=0
```
Comment (googlabati) [inline 11/aashu.config:41]: clock

File: 11/aashu.config [hunk 18]
```diff
@@ -0,0 +1,125 @@
...
+
+
+# Set Pixelize=0 if you want to skip installing all packages belonging to Pixelize Package
+Pixelize=0
+>>PixelSetupWizard=1
+>>AndroidMigratePrebuilt=1
+>>PixelTips=1
+>>PixelConfigOverlays=1
+
+GoogleCalculator=0
```
Comment (googlabati) [inline 11/aashu.config:57]: calculator

File: 11/aashu.config [hunk 19]
```diff
@@ -0,0 +1,125 @@
...
+Drive=0
```
Comment (googlabati) [inline 11/aashu.config:58]: drive

File: 11/aashu.config [hunk 20]
```diff
@@ -0,0 +1,125 @@
...
+Gmail=0
+GooglePhotos=0
+DeviceHealthServices=0
+Flipendo=0
+Velvet=0
+GBoard=0
+
+# Set PixelLauncher=0 if you want to skip installing all packages belonging to PixelLauncher Package
+PixelLauncher=0
+>>PixelLauncher=1
```
Comment (googlabati) [inline 11/aashu.config:70]: pixel launcher

File: 11/aashu.config [hunk 21]
```diff
@@ -0,0 +1,125 @@
...
+>>DevicePersonalizationServices=1
+>>QuickAccessWallet=1
+
+
+# Set GoogleFiles=0 if you want to skip installing all packages belonging to GoogleFiles Package
+GoogleFiles=0
+>>GoogleFiles=1
```
Comment (googlabati) [inline 11/aashu.config:77]: files

File: 11/aashu.config [hunk 22]
```diff
@@ -0,0 +1,125 @@
...
+>>StorageManager=1
```
Comment (googlabati) [inline 11/aashu.config:78]: StorageManager

File: 11/aashu.config [hunk 23]
```diff
@@ -0,0 +1,125 @@
...
+
+GoogleRecorder=0
+GoogleCalendar=1
+MarkupGoogle=0
+GoogleWallpaper=0
+GoogleFeedback=0
+GooglePartnerSetup=0
+GoogleSounds=0
```
Comment (googlabati) [inline 11/aashu.config:86]: sounds

File: 11/aashu.config [hunk 24]
```diff
@@ -0,0 +1,125 @@
...
+AndroidDevicePolicy=0
+GoogleKeep=1
```
Comment (googlabati) [inline 11/aashu.config:88]: GoogleKeep

File: 11/aashu.config [hunk 25]
```diff
@@ -0,0 +1,125 @@
...
+Books=0
+Assistant=0
```
Comment (googlabati) [inline 11/aashu.config:90]: Assistant

File: 11/aashu.config [hunk 26]
```diff
@@ -0,0 +1,125 @@
...
+YouTubeMusic=0
+PlayGames=0
+GoogleTTS=1
+GoogleTalkback=0
+DeviceSetup=0
+AndroidAuto=0
+
+# Set GoogleChrome=0 if you want to skip installing all packages belonging to GoogleChrome Package
+GoogleChrome=0
```
Comment (googlabati) [inline 11/aashu.config:99]: chrome

File: 11/aashu.config [hunk 27]
```diff
@@ -0,0 +1,125 @@
...
+>>GoogleChrome=0
```
Comment (googlabati) [inline 11/aashu.config:100]: chrome

**Review by googlabati — COMMENTED** [review]
me noom sir

**Comment by aashu** [comment]
> me noom sir

What?

**Pull request merged**